### ISSUE TITLE
`display` should not allow an arbitrary string

### ISSUE THREAD

**Issue by pelotom** [frenic/csstype#8]
What is the `| string` for here?

https://github.com/frenic/csstype/blob/e48d1c5c08896a8555829d76ba5ee9f022f7c964/index.d.ts#L1831

**Comment by frenic** [comment]
According to [spec](https://developer.mozilla.org/en-US/docs/Web/CSS/display#Formal_syntax), the syntax tells us that you're able to use space separated values. So fallback to `string` is necessary. But a goal I have is that `display` won't do that so I'm currently experimenting of combining data types with keywords to prevent that. So the result will be like: `'block' | 'flow' | 'block flow' | 'flow block' | ...`

**Comment by pelotom** [comment]
Ah, I never knew `display` supported multiple values.

**Comment by ben-x9** [comment]
I was about to post the same issue for `alignItems` and `justifyContent`. I take it these need to support multiple values too?

**Comment by ben-x9** [comment]
Exposing an `AlignItemsSingle` type would be useful for disabling multiple values in certain contexts where you're sure you don't need it. E.g.:

```
export type AlignItemsSingleProperty = Globals | SelfPosition | "baseline" | "normal" | "stretch";
export type AlignItemsProperty = AlignItemsSingleProperty | string;
```

**Comment by frenic** [comment]
I've started working on combined keywords #22 but halted partly due to #51. I'm afraid that it may have impact on performance if unions increase to an unsustainable amount. I could however limit that some how if that becomes a problem.

Too many type aliases seems to have an impact on performance too. So I'm a bit conservative to splitting them up.

I'll try to resume my work on combined keywords. To at least try it out. But I haven't come that far so anyone is welcome to submit a PR.

**Comment by frenic** [comment]
Fixed in #74
